src/main/java/com/openlattice/chronicle/services/ChronicleServiceImpl.java

```diff
@@ -143,8 +148,9 @@ public ChronicleServiceImpl(
                 .build( new CacheLoader<Class<?>, ApiClient>() {
                     @Override
                     public ApiClient load( Class<?> key ) throws Exception {
+
                         String jwtToken = MissionControl.getIdToken( username, password );
-                        return new ApiClient( () -> jwtToken );
+                        return new ApiClient ( () -> jwtToken );
```

Comment:
also super minor, but can you remove this space? (Not that this breaks anything, but it makes it much easier when reading through this to tell that ApiClient is being instantiated here)